content/posts/hello-world-dbanalytics.md

```diff
@@ -1,13 +1,45 @@
 ---
 author: "Dylan Baker"
-date: "2019-01-05"
-title: "Hello, World: DBAnalytics"
+date: "2019-01-14"
+title: "Hello, World: Becoming a Freelance Consultant"
 description: "Why I'm moving into freelance consulting."
-draft: true
+draft: false
 ---
 
-At the end of next week, I will leave my full-time job as BI Lead at Growth Street. 
+At the end of this week, I'll be leaving my full-time job as BI Lead at Growth Street. I'm incredibly excited to be making the move into freelance consulting.
 
+I'm writing this quick post to talk a bit about why I'm making the move, as much for my own benefit as that of the three other people who will read this blog (hi mum 👋).
 
-![Hope this works!](/test-image.jpg)
+## Why Now
 
+The analytics toolkit available to early-stage businesses has completely changed over the past five years. Five years ago, building out an analytics function was an expensive and difficult undertaking. [It required a fair bit of data engineering](https://blog.fishtownanalytics.com/does-my-startup-data-team-need-a-data-engineer-b6f4d68d7da9). It required building a lot of things from the ground up, well before you saw any value from it.
```

Comment:
I feel like you can replace the second "Five years ago" with something like "back then". Otherwise it just feels a bit repetitive.